lib/ansible/modules/command.py

```diff
@@ -161,6 +161,17 @@
       - dbname with whitespace
     creates: /path/to/database
 
+- name: Run command using argv with mixed argument formats
+  ansible.builtin.command:
+    argv:
+      - /path/to/binary
+      - -v
+      - --debug
+      - --longopt
+      - value for longopt
```

Comment:
```suggestion
      - value_for_longopt
```

Reply:
This is specifically using spaces, to show that in argv an argument with spaces is considered to be a single argument, and not multiple arguments.

Reply:
TIL. Thanks for the explanation.